### PR TITLE
Add Debian package configuration

### PR DESCRIPTION
**Pull request by joshuaboniface**
Adds a simple Debian packaging configuration, following the model of `jellyfin-web`. Once installed it can be enabled by overriding the `JELLYFIN_WEB_OPT=` configuration to point to the new folder `/usr/share/jellyfin/vue`.

Not yet integrated into CI since I'll have to figure that out at a later time.

### CODE REVIEW

File: debian/po/templates.pot [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+"Project-Id-Version: jellyfin-server\n"
+"Report-Msgid-Bugs-To: jellyfin-server@packages.debian.org\n"
+"POT-Creation-Date: 2015-06-12 20:51-0600\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
```
Comment (ferferga) [inline debian/po/templates.pot:18]: This doesn't need to be filed?

File: debian/rules [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	dh $@
+
+# disable "make check"
+override_dh_auto_test:
+
+# disable stripping debugging symbols
+override_dh_clistrip:
+
+override_dh_auto_build:
+	npm install
```
Comment (ferferga) [inline debian/rules:14]:
```suggestion
	npm ci --no-audit
```

**Review by ferferga — COMMENTED** [review]
If possible, please also use conventional commits to have all checks green.

**Comment by joshuaboniface** [comment]
>If possible, please also use conventional commits to have all checks green.

I don't actually know that this means.

File: debian/po/templates.pot (hunk 1 above)
Comment (joshuaboniface) [reply debian/po/templates.pot:18]: It hasn't been on any of our packages so far; AFAIK this is just for "official repository" packages.

**Review by ferferga — APPROVED** [review]

**Pull request merged**